### PR TITLE
fix comment typo

### PR DESCRIPTION
**Pull request by KevinWu0904**

### CODE REVIEW

File: dkron/config.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ type Config struct {
 	AdvertiseRPCPort int `mapstructure:"advertise-rpc-port"`
 
 	// LogLevel is the log verbosity level used.
-	// It cound be (debug|info|warn|error|fatal|panic).
+	// It could be (debug|info|warn|error|fatal|panic).
```
Comment (yvanoers) [inline dkron/config.go:84]: It is supposed to be 'can', actually. 'Could' is hypothetical, which doesn't apply in this case.

**Review by yvanoers — CHANGES_REQUESTED** [review]
Thanks, but let's fix it properly :)

**Comment by KevinWu0904** [comment]
@yvanoers Thanks for your review :)

**Review by yvanoers — APPROVED** [review]

**Comment by yvanoers** [comment]
Thanks @KevinWu0904 !

**Pull request merged**